submissions/madmaxWMFU/google-popup/css/style.css

```diff
@@ -428,3 +427,12 @@ a>.myAccount {
 hr {
     margin: 20px;
 }
+
+ul {
```

Comment:
Avoid styling by tag. Except for the assignment of global base styles.
Example:
```CSS
* {
    box-sizing: border-box;
}

html {
    margin: 0;
    padding: 0;
}

ul {
    list-style: none;
    margin: 0;
    padding: 0;
}
```

Reply:
Done!